backend/kernelCI_app/management/commands/helpers/kcidbng_ingester.py

```diff
@@ -270,11 +257,24 @@ def prepare_file_data(filename, trees_name, spool_dir, io_schema):
         }
 
 
-def db_worker(db_client, stop_event):  # noqa: C901 (too complex)
+def process_submission_item(data: dict[str, Any], metadata: dict[str, Any]):
+    with db_lock:
+        insert_submission_data(data, metadata)
+
+    if VERBOSE and "processing_time" in metadata:
+        ing_speed = metadata["fsize"] / metadata["processing_time"] / 1024
+        logger.info("Ingested %s in %.2f KB/s", metadata["filename"], ing_speed)
+
+
+def db_worker(stop_event: threading.Event):
     """
     Worker thread that processes the database queue.
     This is the only thread that interacts with the database.
+
+    Params:
```

Comment:
I'm not sure about the rest of the repo, but 

```suggestion
    Args:
```

ref: https://sphinxcontrib-napoleon.readthedocs.io/en/latest/example_google.html

Reply:
It seems that the repo it is mixed between the two :melting_face: